app/controllers/v0/letters_generator_controller.rb

```diff
@@ -31,9 +32,7 @@ def index
     def download
       permitted_params = params.permit(DOWNLOAD_PARAMS)
       letter_options =
-        permitted_params.to_h
-                        .select { |_, v| v == 'true' }
-                        .transform_values { |v| ActiveModel::Type::Boolean.new.cast(v) }
+        permitted_params.to_h.transform_keys { |k| k.camelize(:lower) }
```

Comment:
What changed to where we're guaranteed the param options are boolean `true/false`?

Reply:
@kjduensing after switching to POST, they don't come in as strings anymore, but true booleans!

Reply:
Oh interesting, thanks